src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -69,22 +73,25 @@ export class InstructorSearchPageComponent implements OnInit {
    * Searches for students and questions/responses/comments matching the search query.
    */
   search(searchQuery: SearchQuery): void {
+    this.loadingBarService.showLoadingBar();
```

Comment:
So every time I make a request I shall call `this.loadingBarService.showLoadingBar();` and then `this.loadingBarService.hideLoadingBar()`?

Reply:
Yes.

Reply:
Why no inject it to httpservice?

Reply:
I misunderstood the initial question. The above methods are only called on certain request calls and not on all.